dq-game/src/server/controller.py

```diff
@@ -1,26 +1,100 @@
-import sys
 import os
-from utils.socket_connection import ServerSocketConnection
+import pickle
+import sys
+
 from events.event_handler import EventHandler
+from utils.socket_connection import ServerSocketConnection
 
 
 class Controller(EventHandler):
-    def __init__(self, no_of_players):
+    def __init__(self, socket, no_of_players):
         """
         Creates a controller
         """
+        self.socket = socket
+        self.socket.on("game_ready_to_start", self.start)
         self.no_of_players = no_of_players
-        self.socket = ServerSocketConnection(8000)
-        self.socket.on("game_ready_to_start", self.start_game)
+        self.players = []
+
+    def start(self):
+        self.trigger("start_game")
+
+    def request_theme_choices(self, theme_list):
+
+        self.socket.send_to_all(theme_list, "data-theme-list")
```

Comment:
I send a theme list (which we'll be getting from the Back Office) and then I send the `CHOOSE THEME` event to trigger client behavior